projects/demo/src/modules/components/carousel/carousel.template.html

```diff
@@ -130,6 +131,18 @@
                 Current index
             </ng-template>
         </tui-doc-documentation>
+        <tui-doc-documentation heading="CSS customization">
+            <ng-template
+                i18n
+                documentationPropertyMode="input"
+                documentationPropertyType="string"
+                documentationPropertyName="style.--tui-carousel-item-padding"
```

Comment:
Let's rename it to just `--tui-carousel-padding`, that name is rather long and it should be obvious it's not external padding anyway.

Reply:
fixed